lib/errorSerialization.js

```diff
@@ -0,0 +1,18 @@
+var serializeError = require('serializerr');
+var deserializeError = require('deserialize-error');
+
+exports.serializeError = function (error) {
```

Comment:
Did you mean JSONSerializer/SourceMapSerializer? Then it makes sense to rename the file to ErrorSerializer and it's methods to serialize/deserialize.